Added: alt attribute to few images (#20041)
mdn/content: files/en-us/web/api/canvas_api/tutorial/using_images/index.md

To really understand what this does, it may help to look at this image:

> ![There is the source image(rhinoceros in water) on the top left corner. It is rectangular and it's top left corner is at co-ordinates (sx,sy) and whose width and height are sWidth and sHeight respectively. The translated image is shown at the bottom right corner. The top-left corner of the image in the destination canvas is at co-ordinates (dx,dy) and it is scaled to the size specified by dWidth and dHeight.](canvas_drawimage.jpg)

Reviewer: ```suggestion
![The source image is rectangular with top left coordinates are sx and sy and the width and height are sWidth and sHeight respectively. That is translated to the destination canvas where the top-left corner is at coordinates dx and dy, maintaining its aspect ration with a width and height of dWidth and dHeight respectively.](canvas_drawimage.jpg)
```
